code/drasil-example/Drasil/GlassBR/Unitals.hs

```diff
@@ -170,14 +170,21 @@ sd_min     = mkQuantDef (unitary "sd_min"
 {--}
 
 glassBRSymbols :: [UnitaryChunk]
-glassBRSymbols = [min_thick, sflawParamK, sflawParamM, demand, load_dur,
+glassBRSymbols = [min_thick, sflawParamK, sflawParamM, demand, lRe, nonFactorL, load_dur,
   eqTNTWeight]
 
-min_thick, sflawParamK, sflawParamM, demand, sdx, sdy, sdz, load_dur,
+min_thick, sflawParamK, sflawParamM, demand, sdx, sdy, sdz, lRe, nonFactorL, load_dur,
   eqTNTWeight :: UnitaryChunk
 
 demand      = unitary "demand"      (nounPhraseSP "applied load (demand)")
   lQ pascal Rational --correct Space used?
+  
+lRe      = unitary "lRe"      (nounPhraseSP "Load resistance")
```

Comment:
The problem might originate here -- our definitions all use lower-case only, and then combinators capitalize when needed.